File: project-euler/001/001.py

def find_natural_numbers(n):
    i=1
    sum=0
    while(i<n):
        if((i % 3) == 0):
            sum = sum + i
        elif ((i % 5) ==0):
            sum = sum +i
        i =i+1
    return sum


if __name__ =="__main__":
    print (find_natural_numbers(1000))

File: project-euler/001/001.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
class Solution {

    static void Main(String[] args) {
        int t = Convert.ToInt32(Console.ReadLine());
        for(int a0 = 0; a0 < t; a0++){
            int n = Convert.ToInt32(Console.ReadLine());
            n=n-1;
           
            long sum = SumDivisbleBy(3,n)+SumDivisbleBy(5,n)-SumDivisbleBy(15,n);
            Console.WriteLine(sum);
            
        }
    }

    static long SumDivisbleBy(int n,int  p){
    long sum = n * ((long)(p / n)) * (((long)(p / n)) + 1) / 2;
    return sum;
}
}

